### DISCUSSION TITLE
I built a MacOS app with whisper.cpp and VAD

### DISCUSSION THREAD

**Discussion by crisdosaygo** [ggml-org/whisper.cpp#544] [Show and tell]
I really like to make voice memos, and think having some transcripts will be useful.

I put it on [the app store](https://apps.apple.com/us/app/wisprnote/id1671480366?mt=12)

It's also on [Show HN](https://news.ycombinator.com/show) now. (exact link: https://news.ycombinator.com/item?id=34964301)

Also [WisprNote on Product Hunt](https://www.producthunt.com/posts/wisprnote)

<img width="1440" alt="Ulysses Hero Shot" src="https://user-images.githubusercontent.com/22254235/221857821-e49020f9-02b4-43c4-9052-f1901581f400.png">

I want to thank @ggerganov who [replied to my comment about prompt settings](https://github.com/ggerganov/whisper.cpp/issues/256#issuecomment-1430100502) which helped me understand his work more.

**Comment by crisdosaygo** [comment]
Funny over on the Silero VAD discussion I see folks worrying that Whisper.cpp is slow. Actually is super fast! I don't know why they have this out-dated or incorrect perspective on it. Speed is not an issue at all with Whisper.cpp. The @ggerganov  is the bomb! Major good work! ❤️

**Comment by niklasmato** [comment]
How did you convert Silero Vad to coreml? I've been trying to get this in coreml voor quite some time now.

**Reply by crisdosaygo** [reply]
I can't remember now, but I recall that part was complex. IIRC I ended up using ONNX? I think. Best of luck with that!! 😆
